posts/2020-10-21-memory-footprint-throughput.adoc

```diff
@@ -15,6 +15,8 @@ Graham Charters <https://github.com/gcharters>
 :url-prefix:
 :url-about: /
 
+**NOTICE**: _This blog post is superseded by a post with more recent performance data.  Use the following link to access the blog post that uses the newer performance data._  link:{url-prefix}/blog/2022/10/17/memory-footprint-throughput-update.html[How does Open Liberty’s performance compare to other cloud-native Java runtimes]
```

Comment:
The way this renders with bold, followed by italics, followed by blue link looks a bit odd, but not actually an issue (especially since this is an old blog) just something to consider for future:
![image](https://user-images.githubusercontent.com/3076261/201148771-62952061-0916-48e5-beb6-c63cc6381631.png)

Reply:
The bold and italic was set by the AsciiDoc syntax, so changing it is straightforward. 
Would this AsciiDoc styling be more fitting for a `NOTICE`?
 -  https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#ex-admon-para

Reply:
I thought about initially, but ended up not putting in my comment because "NOTE" tends to be more of a minor fyi, and this NOTICE is meant to be far more visible then that.
Aka, usually we use a NOTE at the bottom of a section to clarify something minor, and the intent of this NOTICE is quite the opposite.

Reply:
Here's an example of what NOTE looks like:
![image](https://user-images.githubusercontent.com/3076261/201188148-d2e1ae4f-a40b-4471-aec2-5236994500b9.png)